src/main/java/kineticrevolution/blocks/BlockDuster.java

```diff
@@ -104,8 +105,4 @@ public void handleOutputs(World world, int x, int y, int z, List<ItemStack> outp
 		Utils.dropItemstacks(world, x + .5, y + .5, z + .5, outputs);
 	}
 
-	@Override
-	public int damageDropped(int meta) {
-		return 0;
```

Comment:
this is required, i'll re-add this, if people break a duster mid-dusting then they should get one with meta 0 as the meta is the progress